### PR TITLE
RIFEX-68 Refactor rif.config.js to be stored on nifty store instead of file

### PR DESCRIPTION
**Pull request by JONAF2103**
This PR introduces the ability to change the rif configuration on runtime using stores instead of the actual file, this let's us to change the configuration by selected chain on real time.

* Adding a provider for the configuration
* Updating rif controller with the new provider
* Creating logic to store the configuration by chain id
* Integrating provider with rif controller and exposing operations to the ui

Here are some screenshots of the configuration page:

imagen

imagen

imagen

This affects ui and service layers.

### CODE REVIEW

File: app/scripts/controllers/rif/configuration/index.js [hunk 1]
```diff
@@ -20,12 +24,13 @@ export class RifConfigurationProvider {
    */
   loadConfiguration (chainId) {
     const actualState = this.store.getState();
-    if (!actualState.rifConfiguration) {
-      actualState.rifConfiguration = this.getInitialConfigStructure(chainId);
+    console.log('DEBUGGGGGGGG', JSON.stringify(actualState.rifConfiguration));
```
Comment (marcosmartinez7) [inline app/scripts/controllers/rif/configuration/index.js]: Remove this

File: app/scripts/controllers/rif/configuration/index.js [hunk 2]
```diff
@@ -163,13 +168,13 @@ export class RifConfigurationProvider {
   }
 
   getConfigurationObject () {
-    return this.store.getState().rifConfiguration;
+    return this.store.getState().configuration[this.network ? this.network.id : '33'];
```
Comment (marcosmartinez7) [inline app/scripts/controllers/rif/configuration/index.js]: 33 is not so standard for regtest. Lets put that number into a constant so its configurable

File: app/scripts/controllers/rif/configuration/index.js [hunk 3]
```diff
@@ -163,13 +168,13 @@ export class RifConfigurationProvider {
...
   }
 
   setConfigurationObject (configuration) {
     const actualState = this.store.getState();
     if (this.validateConfiguration(configuration)) {
-      actualState.rifConfiguration = configuration;
+      actualState.configuration[this.network ? this.network.id : '33'] = configuration;
```
Comment (marcosmartinez7) [inline app/scripts/controllers/rif/configuration/index.js]: same here

**Review by marcosmartinez7 — CHANGES_REQUESTED** [review]
lgtm in general, just some minor changes

**Review by rorolopetegui — CHANGES_REQUESTED** [review]
lgtm just fix what @marcosmartinez7 requested

File: app/scripts/controllers/rif/configuration/index.js (hunk 1 above)
Comment (JONAF2103) [reply app/scripts/controllers/rif/configuration/index.js]: Removed!

File: app/scripts/controllers/rif/configuration/index.js (hunk 2 above)
Comment (JONAF2103) [reply app/scripts/controllers/rif/configuration/index.js]: Ok didn't knew that, good to know

**Review by marcosmartinez7 — APPROVED** [review]

**Review by rorolopetegui — APPROVED** [review]
lgtm

**Pull request merged**